run/rails-cat_album/config/environments/development.rb

```diff
@@ -33,7 +33,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :google
```

Comment:
It would be better if we could set this to `:local` for development and `:google` for production, so you don't have to hit the google service when testing locally in the development environment. (That would require getting a suitable helper like `url_for` working properly in the templates above. So if it's not immediatly obvious how to get that to work, we can defer it to a follow-up PR. But I do think we need to get it working before we publish.)

Reply:
good call! For the url_for helper, is that referring to the local development testing?

Reply:
Yes: in the view, if storage uses `:local` for development, it stores image data on the local disk rather than in cloud storage, so the img tag needs to be different. I'm assuming that `url_for` knows how to generate the correct URL based on the ActiveStorage service (and if not, then the ActiveStorage documentation/guide should tell us how to generate proper views that will work both for development and production.)